zio-s3/src/main/scala/zio/s3/settings.scala

```diff
@@ -40,10 +40,14 @@ object S3Region { self =>
     new S3Region(Region.of(r)) {}
 }
 
-final case class S3Settings(s3Region: S3Region, credentials: AwsCredentials)
+final case class S3Settings(s3Region: S3Region, credentials: AwsCredentials, forcePathStyle: Option[Boolean])
```

Comment:
i dont think we need to add this parameter to the settings

Reply:
Fixed